Felienne/hedy
# Adding blur button when not in code editor

**Description**

Displays a blur button in the coding interface for students to be able to hide their code when showing it to others. This button is only displayed when not in edition mode so as not to clutter the UI.

**Fixes _issue or discussion number_**

Fixes #3250 

**How to test**

1. Navigate to http://0.0.0.0:8080/hedy#default
2. In this first interface, you shouldn't be able to see any blur button at the bottom right corner of the code editor
3. Follow the level instructions and complete the exercise
4. "Hit Save & share code"
5. Visit the URL in your clipboard and ensure this time there is a blur button on the bottom right corner of the area showing your code
6. Hitting it should blur your code
7. If you click the "Edit code" button again, you should be back to the code editor interface and the blur button should have disappeared.
